docs/pages/debugging/tools.mdx

```diff
@@ -36,7 +129,9 @@ The Developer menu provides multiple options:
 
 Now let's explore some of the more exciting functionalities.
 
-#### Debug Remote JS
+### Debug Remote JS
```

Comment:
This section doesn't make much sense or relevant (trying to find technical accuracy after this comment [here](https://github.com/expo/expo/pull/22787/files#diff-6edc759c19b235dc978c2960b5dcd0d59782b42d45700b1b46c8ec604f423831R83:~:text=Here%20is%20the%20latest%20screenshot).

Or for apps not using Hermes, it still shows Debug Remote JS?